deps.ffmpeg/70-libdatachannel.zsh

```diff
@@ -0,0 +1,112 @@
+autoload -Uz log_debug log_error log_info log_status log_output
+
+## Dependency Information
+local name='libdatachannel'
+local version='0.19.0'
+local url='https://github.com/paullouisageneau/libdatachannel.git'
+local hash='004c6b74ffa1d2718499b84d5bd7674d94b83dbc'
```

Comment:
Are there any newer commits that we'd like to use or is this the latest we're comfortable using?

Reply:
This is the latest commit/one we want

Reply:
It's not the latest commit: https://github.com/paullouisageneau/libdatachannel/compare/004c6b74ffa1d2718499b84d5bd7674d94b83dbc...0abde70beb1310bd393af6cddd6d1a695fe88131

If it's "the one we want for now" because it's what was tested against, then that's fine. We'll likely update it before the next major/minor release of OBS anyway, as we update dependencies when possible for major/minor releases of OBS.

Reply:
@RytoEX I am not sure what to respond with here? This is the commit I would prefer to use.

Reply:
I assume it's simply both of us misunderstanding the other. I intended to ask, "Are there any newer commits that we'd like to use? If not, is it because this is the latest commit we're comfortable using?"

I read your answer as, "This is the latest commit or the one we want," so I rebutted the idea that it's the latest commit, and then attempted to clarify my position that if the reason this commit is chosen is simply "because it's the one we want", then that's fine.

Reply:
Sorry my misunderstanding. 

This is the commit we want. I haven't audited/checked the commits that have arrived after this. I am not comfortable including other commits.

Reply:
Then we'll keep this commit for now. As mentioned, we'll likely update deps if they have updates available when we prep the next deps update for the next major/minor release of OBS Studio.